collector/src/bin/rustc-perf-collector/main.rs

```diff
@@ -174,16 +174,18 @@ fn bench_commit(
             continue;
         }
 
-        let result = benchmark.measure(build_kinds, run_kinds, rustc_path, cargo_path, iterations);
-
-        if let Err(ref s) = result {
-            info!("failed to benchmark {}, recorded: {}", benchmark.name, s);
-        }
+        let mut processor = execute::MeasureProcessor::new(&benchmark.name);
+        let result = benchmark.measure(&mut processor, build_kinds, run_kinds, rustc_path,
+                                       cargo_path, iterations);
+        let result = match result {
+            Ok(()) => Ok(processor.collected),
```

Comment:
This feels quite odd: normally, I'd expect collected to be in the result itself, but I think changing that's a larger refactor than is necessary, and is rather hard. Let's leave it for now.